pyuploadcare/__init__.py

```diff
@@ -15,7 +15,7 @@
 
 from __future__ import unicode_literals
 
-__version__ = '2.1.0'
+__version__ = '3.0.0'
```

Comment:
just revert the change. the scope of this PR is adding two new methods to File resource, that's it.